packages/drivers/debugger/src/sanitizer.ts

```diff
@@ -152,6 +140,7 @@ class ChunkedOpProcessor {
             assert(stringified.length <= this.concatenatedLength);
         } catch (e) {
             console.error(e);
```

Comment:
I'd remove console.error - this is never correct with one exception - top level function in some tool. I.e. libraries should never write to console.

Reply:
removing for debugger ui, keeping for command line